scipy/fft/_basic.py

```diff
@@ -388,6 +388,120 @@ def irfft(x, n=None, axis=-1, norm=None, overwrite_x=False):
     return _pocketfft.irfft(x, n, axis, norm, overwrite_x)
 
 
+def hfft(x, n=None, axis=-1, norm=None, overwrite_x=False):
+    """
+    Compute the FFT of a signal that has Hermitian symmetry, i.e., a real
+    spectrum.
+
+    Parameters
+    ----------
+    x : array_like
+        The input array.
+    n : int, optional
+        Length of the transformed axis of the output. For `n` output
+        points, ``n//2 + 1`` input points are necessary.  If the input is
+        longer than this, it is cropped.  If it is shorter than this, it is
+        padded with zeros. If `n` is not given, it is taken to be ``2*(m-1)``
+        where ``m`` is the length of the input along the axis specified by
+        `axis`.
+    axis : int, optional
+        Axis over which to compute the FFT. If not given, the last
+        axis is used.
+    norm : {None, "ortho"}, optional
+        Normalization mode (see `fft`). Default is None.
+    overwrite_x : bool, optional
+        If True, the contents of `x` can be destroyed; the default is False.
+        See `fft` for more details.
+    Returns
+    -------
+    out : ndarray
+        The truncated or zero-padded input, transformed along the axis
+        indicated by `axis`, or the last one if `axis` is not specified.
+        The length of the transformed axis is `n`, or, if `n` is not given,
+        ``2*m - 2`` where ``m`` is the length of the transformed axis of
+        the input. To get an odd number of output points, `n` must be
+        specified, for instance as ``2*m - 1`` in the typical case,
+    Raises
+    ------
+    IndexError
+        If `axis` is larger than the last axis of `a`.
+    See also
+    --------
+    rfft : Compute the one-dimensional FFT for real input.
+    ihfft : The inverse of `hfft`.
+    hfftn : Compute the n-dimensional FFT of a Hermitian signal.
+    Notes
```

Comment:
It's good practice to check the Sphinx renderings locally (in `doc` do `make html-scipyorg`) or on CircleCI once it completes. You'll see for example that the lack of newlines between these sections is problematic as they are not rendered as section headings:

https://13886-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.fft.hfft.html#scipy.fft.hfft

Reply:
Interesting, I copied those docs straight from numpy so I assumed they were safe. I guess my editor removed the extra whitespace at some point.